internal/test/checks/checks.go

```diff
@@ -53,3 +56,103 @@ func ErrorIsNotf(t *testing.T, err, target error, format string, msg ...string)
 		t.Fatalf(format, msg)
 	}
 }
+
+type TestingT interface {
+	Fatalf(format string, args ...any)
+	Errorf(format string, args ...any)
+}
+
+type tHelper interface {
+	Helper()
+}
+
+// Equal asserts that two objects are equal.
+//
+//	assert.Equal(t, 123, 123)
+//
+// Pointer variable equality is determined based on the equality of the
+// referenced values (as opposed to the memory addresses). Function equality
+// cannot be determined and will always fail.
+func Equal(t TestingT, expected, actual interface{}) bool {
```

Comment:
Could you please change `interface{}` to `any` across the PR?

Reply:
OK